### ISSUE TITLE
[Bug][Accessibility] Informative warning icon is cut off when text spacing is applied

### ISSUE THREAD

**Issue by jamiegaskell** [scottish-government-design-system/design-system#142]
## Description of the issue

When text spacing is applied, the informative warning icon is partially cut off. 

## Steps to reproduce the issue

1. Set the window size to 1920 x 1080px.
2. Apply text spacing using a bookmarklet such as this: [Text spacing bookmarklet](https://codepen.io/stevef/pen/YLMqbo/)

Otherwise the following styling should be applied:

Line height (line spacing) to at least 1.5 times the font size;
Spacing following paragraphs to at least 2 times the font size;
Letter spacing (tracking) to at least 0.12 times the font size;
Word spacing to at least 0.16 times the font size.

See [1.4.12 Text Spacing](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html).

## Expected behaviour

No informative content should be lost when text spacing is applied.

## Screenshots

![The warning icon is partially cut off with text spacing applied](https://github.com/user-attachments/assets/1f08c9b5-d2b0-40fb-8d35-777cb0f07845)

## Environment

- Operating system: Windows 10
- Browser: Edge

**Comment by jamiegaskell** [comment]
Hi @jsutcliffe, has there been any updates on this at all?

**Comment by dkindnes** [comment]
We have a prereleased version `3.0.0-beta.0` which includes the fix for this issue.

**Comment by jsutcliffe** [comment]
Now fully released in v3.0.0.
